drivers/storage/portworx/components_test.go

```diff
@@ -2677,6 +2677,15 @@ func TestAutopilotWithTLSEnabled(t *testing.T) {
 				},
 				TLS: &corev1.TLSSpec{
 					Enabled: boolPtr(true),
+					RootCA: &corev1.CertLocation{
+						FileName: stringPtr("/etc/pwx/ca.crt"),
+					},
+					ServerCert: &corev1.CertLocation{
+						FileName: stringPtr("/etc/pwx/server.crt"),
+					},
+					ServerKey: &corev1.CertLocation{
+						FileName: stringPtr("/etc/pwx/server.key"),
+					},
```

Comment:
In case of filenames, we expect the CA cert to be part of default secret as well for autopilot?
If that is the expectation, then why not have a default CA cert for portworx as well? Why expect the user to have both the secret and filename?

Reply:
> In case of filenames, we expect the CA cert to be part of default secret as well for autopilot?
> If that is the expectation, then why not have a default CA cert for portworx as well? Why expect the user to have both the secret and filename?

That is a valid point. Yes that is extra work for the user if they choose to use a file.... but that's a choice. The customer has 2 choices

Choice 1: Customer places the CA cert in a k8s secret of their choice - they're done, we do not need them to create a `px-api-root-ca` k8s secret
^^^^^^^
I make code change based on your feedback to support this.

Choice 2: user chooses to create a CA file and install it on the node. Now they have to do extra work to also upload this into a k8s secret named `px-api-root-ca` for clients to access.
Maybe in the future we can figure out a way to make the choice2 experience better.